src/components/ec-blog-item/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './ec-blog-item.vue';
```

Comment:
Don't forget to re-export it also in `main.js` so in EBO, we can easily import it in EBO:
```
import { EcBlogItem } from `@ebury/chameleon-components`;
```